eq_bot/game/buff/buff_manager.py

```diff
@@ -13,8 +13,9 @@ def __init__(self, eq_window: EverQuestWindow, guild_tracker: GuildTracker):
         self._guild_tracker = guild_tracker
 
     def handle_tell_message(self, tell_message):
-        # TODO: Enqueue and process messages when window is not busy given that messages will come in asynchronously
+        self._eq_window.handle_window_action(self.handle_tell_message_async(tell_message))
```

Comment:
```suggestion
        self._eq_window.handle_window_action(lambda: self.handle_tell_message_async(tell_message))
```

The handle_window_action function will need to call and enqueue a function.